# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome24/evolution/hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN -x '*~' evolution-1.4.0-old/camel/providers/nntp/camel-nntp-store.c evolution-1.4.0-new/camel/providers/nntp/camel-nntp-store.c
--- evolution-1.4.0-old/camel/providers/nntp/camel-nntp-store.c	2002-05-28 05:00:31.000000000 +0200
+++ evolution-1.4.0-new/camel/providers/nntp/camel-nntp-store.c	2003-07-04 16:51:28.000000000 +0200
@@ -114,7 +114,7 @@
 #ifdef HAVE_SSL
 	if (ssl_mode != USE_SSL_NEVER) {
 		port = service->url->port ? service->url->port : NNTPS_PORT;
-		tcp_stream = camel_tcp_stream_ssl_new (service, service->url->host);
+		tcp_stream = camel_tcp_stream_ssl_new (service, service->url->host, 0);
 	} else {
 		tcp_stream = camel_tcp_stream_raw_new ();
 	}
diff -urN -x '*~' evolution-1.4.0-old/camel/providers/nntp/camel-nntp-summary.h evolution-1.4.0-new/camel/providers/nntp/camel-nntp-summary.h
--- evolution-1.4.0-old/camel/providers/nntp/camel-nntp-summary.h	2002-05-07 09:31:25.000000000 +0200
+++ evolution-1.4.0-new/camel/providers/nntp/camel-nntp-summary.h	2003-07-04 15:13:18.000000000 +0200
@@ -24,7 +24,6 @@
 #include <camel/camel-folder-summary.h>
 #include <camel/camel-folder.h>
 #include <camel/camel-exception.h>
-#include <libibex/ibex.h>
 
 #define CAMEL_NNTP_SUMMARY(obj)         CAMEL_CHECK_CAST (obj, camel_nntp_summary_get_type (), CamelNNTPSummary)
 #define CAMEL_NNTP_SUMMARY_CLASS(klass) CAMEL_CHECK_CLASS_CAST (klass, camel_nntp_summary_get_type (), CamelNNTPSummaryClass)
